PWGLF/Tasks/Resonances/kstarqa.cxx

```diff
@@ -102,6 +102,12 @@ struct Kstarqa {
     Configurable<float> cfgRCRFC{"cfgRCRFC", 0.8f, "Crossed Rows to Findable Clusters"};
     Configurable<float> cfgITSChi2NCl{"cfgITSChi2NCl", 36.0, "ITS Chi2/NCl"};
     Configurable<float> cfgTPCChi2NCl{"cfgTPCChi2NCl", 4.0, "TPC Chi2/NCl"};
+
+    // Other fixed variables
+    float lowpTcutinpTdepPID = 0.5;
```

Comment:
Use proper camelCase. If you break `lowpTcutinpTdepPID` down into words, you get `lowp-tcutinp-tdep-p-i-d` which is not what you mean.